llvm/include/llvm/Analysis/TargetTransformInfoImpl.h

```diff
@@ -363,6 +363,10 @@ class TargetTransformInfoImplBase {
     return 0;
   }
 
+  InstructionCost getPHIScalarizationOverhead(Type *ScalarTy, VectorType *VTy) {
+    return 0;
```

Comment:
For i32/pointer/multiples of 2 x 16 should be 0

Reply:
Outdated